app/Http/Controllers/ParamController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Param;
+use App\Models\Project;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class ParamController extends Controller
+{
+    public function index(Project $project): JsonResponse{
+		return new JsonResponse($project->params);
```

Comment:
On peut retourne une 200 ici 
```suggestion
		return new JsonResponse($project->params, 200);
```